client/src/Components/Login/Login.js

```diff
@@ -23,33 +24,42 @@ const Login = () => {
   }
 
   return (
-    <>
-      <h2> Login to an existing account</h2>
+    <div className="login-container" style={{margin: "auto"}}>
+      <h2 style={{fontSize: "2em", color: "#7D615F"}}>log in</h2>
```

Comment:
we should try to do styles in their own folders, but this is okay for now (i am guilty of this as well)